### PR TITLE
SC21 RSE-HPC workshop blog

### PR DESCRIPTION
**Pull request by bernhold**
# Instructions (DELETE THIS SECTION)

To view these instructions, please hit the **Preview** tab (above left). After you have read all of these instructions, delete them entirely from the body of this issue before entering your submission.

Be sure to select `master` as the `base` branch above as the target for this PR.

Provide a general summary of your changes in the title above.

Fill in the below **Description** section with minimal text describing the changes/new contributions in this PR and replace `<...>` as appropriate.

Remove the beginning `<!--` and ending `-->` comment markers for the correct **checklist** to be used for this PR and delete the other checklist.  (The checklist items for files displayed on the bssw.io site are shown by default.  The checklist items for internal files not displayed in the bssw.io site is given below with commented out checklist items by default.)

Any checklist items that don't apply can be striked out by adding `~~` to the beginning and end of the checklist item as `* ~~[] <checklist-item>~~`.  Also, remove the strikeout markers `~~` for the [wikize_refs.py] checklist items if using formal citations for bssw.io contributions.

# Description

EB Member: @`<eb-member-id>`

Resolves #`<issue-id>`

`<Other minimal information about the PR.>`

## PR checklist for files displayed on bssw.io site

* [ ] ***[Author]*** `@mention` the BSSw.io editorial board member `@<eb-member-id>` in **Description** above assigned to shepherd your PR.
* [ ] ***[Author]*** Add the `<issue-id>` in the **Description** above for the associated GitHub Issue.
* ~~[ ] ***[Author]*** Ensure `wikize_refs.py -i <base>.md` is run and commit (if using [wikize_refs.py])~~.
* [ ] ***[Author]*** Inspect the content in the `*.md` file(s) as rendered in GitHub for this PR.
* [ ] ***[EB Mem]*** Assign this PR to the EB member `<eb-member-id>`.
* [ ] ***[EB Mem]*** Assign this PR to the author of the PR `<pr-author-id>`.
* [ ] ***[EB Mem]*** Add one or more Reviewers.
* [ ] ***[EB Mem]*** Add label `content: <content-type>` for the type of contribution.
* [ ] ***[EB Mem]*** Add to Project `Content Development` (see [Content Development]).
* [ ] ***[EB Mem]*** Add [meta-data] to the `*.md` file(s) (set `Publish: yes`).
* [ ] ***[EB Mem]*** Add label `preview` (so PR branch will be merged to 'preview' branch and watch for possible merge failures).
* [ ] ***[EB Mem]*** Rebuild [preview] site and confirm new content is there, renders correctly and is returned in searches.
* [ ] ***[Author]*** Make any final changes to the PR based on feedback.
* ~~[ ] ***[Author]*** Ensure `wikize_refs.py -i <base>.md` is run and commit (if using [wikize_refs.py]).~~
* [ ] ***[EB Mem]*** Rebuild [preview] site and re-confirm content looks correct.
* [ ] ***[EB Mem]*** Ensure at least one reviewer signs off on the final changes.
* [ ] ***[EB Mem]*** Move the PR to "Ready to Publish" in [Content Development].
* [ ] ***[EB Mem]*** Leave comment and @mention `@betterscientificsoftware/bssw-maint` (**BSSw Maint**) asking to carry out final publication steps.
* [ ] ***[BSSw Maint]*** Verify that all needed files are present in the PR (article, images, updates to Site/Homepage.md carousel and/or Site/Announcements/Announcements.md as appropriate).
* [ ] ***[BSSw Maint]*** Merge PR. (Should automatically move to "Done" in [Content Development].)
* [ ] ***[BSSw Maint]*** Verify new contribution shows up on [bssw.io] as expected.

NOTE:
* Checklist items prefixed with ***[Author]*** are expected to be performed by the author of the PR or can be performed by the author.
* Checklist items prefixed with ***[EB Mem]*** must be performed by a [BSSw.io Editorial Board (EB) Member](https://betterscientificsoftware.github.io/bssw.io/bssw_members.html).
* Checklist items prefixed with ***[BSSw Maint]*** must be performed by someone in the `@betterscientificsoftware/bssw-maint` team (hint: type `@`, `b`, `s`, `s`, `w`, `-`, `m`  to auto-complete to `@betterscientificsoftware/bssw-maint`).

## PR checklist for (internal) files not displayed on bssw.io site

*Click "Write" above and remove comment markers to see below checklist items*

[preview]: https://preview.bssw.io
[bssw.io]: https://bssw.io
[Content Development]: https://github.com/betterscientificsoftware/bssw.io/projects/3
[BSSw Internal]: https://github.com/betterscientificsoftware/bssw.io/projects/2
[meta-data]: https://betterscientificsoftware.github.io/bssw.io/bssw_styling_common.html#metadata-section
[wikize_refs.py]: https://github.com/betterscientificsoftware/bssw.io/blob/master/utils/README.md#wikize_refspy

### CODE REVIEW

File: Articles/Blog/2022-03-sc21-rse-workshop.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+community collaboration and shared policies, while advancing scientific
+productivity through better scientific software. She concluded her talk with
+a shout-out to her ECP RSE colleagues at Argonne and elsewhere, describing
+them as "the linchpin of all things exascale - and beyond"!
+
+### Short talks
+
+Marion Weinzierl gave the first short talk on "Training HPC RSEs: What we
+learned from our performance." This was based on training workshops that
+they gave, with seven sessions over seven months. These lessons were
```
Comment (karbarz) [inline Articles/Blog/2022-03-sc21-rse-workshop.md:43]: Maybe I missed this, but who is "they" in this sentence? Maybe clairfy?

File: Articles/Blog/2022-03-sc21-rse-workshop.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+technical and sociological challenges. In response, ECP has developed an
+open, hierarchical software ecosystem, building a "team of teams" based on
+community collaboration and shared policies, while advancing scientific
+productivity through better scientific software. She concluded her talk with
+a shout-out to her ECP RSE colleagues at Argonne and elsewhere, describing
+them as "the linchpin of all things exascale - and beyond"!
+
+### Short talks
+
+Marion Weinzierl gave the first short talk on "Training HPC RSEs: What we
```
Comment (karbarz) [inline Articles/Blog/2022-03-sc21-rse-workshop.md:41]:
```suggestion
Marion Weinzierl gave the first short talk on "Training HPC RSEs: What We
```

File: Articles/Blog/2022-03-sc21-rse-workshop.md [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+learned from our performance." This was based on training workshops that
```
Comment (karbarz) [inline Articles/Blog/2022-03-sc21-rse-workshop.md:42]:
```suggestion
Learned From Our Performance." This was based on training workshops that
```

File: Articles/Blog/2022-03-sc21-rse-workshop.md [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+Software Engineers" was presented by Reed Milewicz. The talk discussed
+mentorship as a strategy to enable the career growth and retention of RSEs;
+key needs that RSEs may have in providing and receiving mentorship; and
+directions for future work. The talk defined mentorship as a relationship in
+which a more experienced or more knowledgeable person (a mentor) helps to
+guide a less experienced or less knowledgeable person (a mentee), and talked
+about the benefits for both the mentee and mentor. Milewicz found that RSEs
+need interdisciplinary mentorship networks, long-term mentoring
+relationships, and training in soft skills. The talk called for us to start
+mentorships now, to build explicit institutional support for them, and to
```
Comment (karbarz) [inline Articles/Blog/2022-03-sc21-rse-workshop.md:74]:
```suggestion
mentorships now, to build explicit institutional support for these mentorships, and to
```

File: Articles/Blog/2022-03-sc21-rse-workshop.md [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+be used to measure progress throughout an RSE career. To support this work,
+it was noted that there must be further progress on stabilizing RSEs within
+the academia, and that this work would be advanced if parallel efforts to
+recognize software as a vital research output are successful.
+
+### Panel discussion: What role(s) do RSEs play in the HPC community? What kind of support do they need?
+
+Five panelists from diverse backgrounds - academia, national lab, industry -
+and HPC experience discussed the unique challenges and roles RSEs have in
+HPC being themselves RSEs or supporting building career paths for RSEs.
```
Comment (karbarz) [inline Articles/Blog/2022-03-sc21-rse-workshop.md:125]:
```suggestion
HPC, being themselves RSEs, or supporting building career paths for RSEs.
```

File: Articles/Blog/2022-03-sc21-rse-workshop.md [hunk 6]
```diff
@@ -0,0 +1,204 @@
...
+Efficient parallelization is hard in HPC and while RSEs aim at following
```
Comment (karbarz) [inline Articles/Blog/2022-03-sc21-rse-workshop.md:126]:
```suggestion
Efficient parallelization is hard in HPC and, while RSEs aim at following
```

File: Articles/Blog/2022-03-sc21-rse-workshop.md [hunk 7]
```diff
@@ -0,0 +1,204 @@
...
+good or best practices in software engineering, there is a lack of available
+literature and practices in HPC yet. Tools supporting software engineering
+often do not work well in an exascale environment and the whole programming
+paradigm changes. The panelists agreed that the role of HPC RSEs is still
+missing good career paths in academia and that their expertise is often not
+acknowledged. The panelists have different experiences with support in HPC
+centers - some experienced good support with training and incentives and
```
Comment (karbarz) [inline Articles/Blog/2022-03-sc21-rse-workshop.md:133]:
```suggestion
centers - some experienced good support with training and incentives, while others
```

File: Articles/Blog/2022-03-sc21-rse-workshop.md [hunk 8]
```diff
@@ -0,0 +1,204 @@
...
+some have seen that it got better over the last few years but was also a
```
Comment (karbarz) [inline Articles/Blog/2022-03-sc21-rse-workshop.md:134]:
```suggestion
have seen that it got better over the last few years, but was also a
```

File: Articles/Blog/2022-03-sc21-rse-workshop.md [hunk 9]
```diff
@@ -0,0 +1,204 @@
...
+### Conversation: RSEs and HPC in China
+
+The conversation with James Lin, Vice Director of the HPC center of the
+Shanghai Jiao Tong University, gave insights into the HPC ecosystem and that
+software has been added to the focus besides hardware and applications in
+HPC. The support of self-developed domestic software such as molecular
+simulations and optimizations of software has started to be especially
+supported via HPC centers. Research software is developed in China via
+universities, research institutes, companies, and HPC centers. Building out
+RSE teams is not very common in China yet though and to the best of his,
```
Comment (karbarz) [inline Articles/Blog/2022-03-sc21-rse-workshop.md:151]:
```suggestion
RSE teams is not very common in China yet though and to the best of his knowledge, 
```

File: Articles/Blog/2022-03-sc21-rse-workshop.md [hunk 10]
```diff
@@ -0,0 +1,204 @@
...
+simulations and optimizations of software has started to be especially
+supported via HPC centers. Research software is developed in China via
+universities, research institutes, companies, and HPC centers. Building out
+RSE teams is not very common in China yet though and to the best of his, 
+his university 
+currently has the only dedicated RSE team in China. He is one of the
+advocates for the role and profession in China and aims at starting with a
+local community. The path to build teams would differ between universities,
+HPC centers and research institutes because of differences in existing
+possibilities like hiring staff for software development and funding
```
Comment (karbarz) [inline Articles/Blog/2022-03-sc21-rse-workshop.md:157]:
```suggestion
possibilities, like hiring staff for software development and funding
```

File: Articles/Blog/2022-03-sc21-rse-workshop.md [hunk 11]
```diff
@@ -0,0 +1,204 @@
...
+Our final session was a panel on building RSE communities worldwide, with
+six panelists from four continents. All of their countries have RSEs and RSE
+activities in some form, though often without using or even knowing the term
+RSE. (One panelist mentioned that she first heard the term RSE when we
+invited her to be on the panel!). The panelists noted several common factors
+that have helped build their communities. Communication channels such as
+mailing lists, Slack, and community calls are very effective ways for RSEs
+to start connecting. Raising RSE awareness in other software or research
+conferences can also build connections and provide a nucleus for new RSE
+communities. Sustainability of newly-forming groups Is a key concern:
```
Comment (karbarz) [inline Articles/Blog/2022-03-sc21-rse-workshop.md:173]:
```suggestion
communities. Sustainability of newly-forming groups is a key concern:
```

File: Articles/Blog/2022-03-sc21-rse-workshop.md [hunk 12]
```diff
@@ -0,0 +1,204 @@
...
+several countries have grown their communities to where they would be
+sustainable beyond the loss of any single person, although perhaps not yet
+beyond the loss of a small core group. And several examples were given
+showing that RSEs could effectively leverage existing community efforts
+among practitioners and students.
+
+### Summary
+
+Overall, the workshop discussions were very active and many good questions
```
Comment (karbarz) [inline Articles/Blog/2022-03-sc21-rse-workshop.md:182]:
```suggestion
Overall, the workshop discussions were very active. Many good questions
```

File: Articles/Blog/2022-03-sc21-rse-workshop.md (hunk 4 above)
Comment (bernhold) [reply Articles/Blog/2022-03-sc21-rse-workshop.md:74]: I think it is okay as is.  We already have the word twice in the same sentence.

File: Articles/Blog/2022-03-sc21-rse-workshop.md (hunk 1 above)
Comment (bernhold) [reply Articles/Blog/2022-03-sc21-rse-workshop.md]: Modified text to address this

**Pull request merged**